pyproject.toml

```diff
@@ -38,8 +38,8 @@ dependencies = [
 ]
 
 [project.urls]
-homepage = "https://github.com/deepmind/optax"
-repository = "https://github.com/deepmind/optax"
+homepage = "https://github.com/google-deepmind/optax"
```

Comment:
As we are on it, shouldn't we change the authors as 
authors = [
    {name = "Google DeepMind", email = "optax-dev@google.com"},
]
?

Reply:
indeed, done!